python/GafferUI/NumericSlider.py

```diff
@@ -128,12 +129,29 @@ def valueChangedSignal( self ) :
 
 		return self.__valueChangedSignal
 
+	def _snapPosition( self, position ) :
+
+		value = self._valueForPosition( position )
+		return self._positionForValue( self._snapValue( value ) )
+
+	def _snapValue( self, value ) :
+
+		return round( value / self._snapIncrement ) * self._snapIncrement
```

Comment:
(Not fully resurfaced all of the reasoning in my head yet but) I guess it seemed like it made sense (to my silly brain at least) to say that when you configure a `Slider`, snap values are in positions, as thats all that they have. When you configure a `NumericSlider`, then they are all in numeric space. Otherwise, afaict, you'd have to make all the `valueToPosition` methods public too (as opposed to the requested even-more-private) so people could set meaningful values. 2p